### PR TITLE
Added DERE DBook D10

### PR DESCRIPTION
**Pull request by junocomp**
Please add this to main

### CODE REVIEW

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -338,6 +338,14 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnDellInc.*:pnVenue8Pro3845:*
 # Dell Venue 10 Pro 5055
 sensor:modalias:acpi:INVN6500*:dmi:*svnDell*:pnVenue10Pro5055:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; 1, 0, 0; 0, 0, 1
+
```
Comment (yuwata) [inline hwdb.d/60-sensor.hwdb]: Please drop the space in this empty line.

**Review by junocomp — COMMENTED** [review]
accepted

**Comment by junocomp** [comment]
any chance of resolving this?

**Comment by yuwata** [comment]
Please address the above comment. And, force-push on your `patch-1` repository.

**Comment by junocomp** [comment]
@yuwata there seems to skipped one

**Pull request merged**